src/color.scss

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2023 Alaska Airlines. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// Import Auro tokens
+@import './../node_modules/@aurodesignsystem/design-tokens/dist/tokens/SCSSVariables';
+
+:host([visible]) {
+  background-color: var(--ds-auro-toast-background-color);
+  color: var(--ds-auro-toast-text-color);
+
+  .icon {
+    color: var(--ds-auro-toast-icon-color);
+  }
+
+  .closeButton {
+    background-color: var(--ds-auro-toast-close-button-background-color);
+    color: var(--ds-auro-toast-close-button-icon-color);
+
+    &:hover {
+      border: 1px solid var(--ds-auro-toast-close-button-border-color);
+      
+      --ds-auro-toast-close-button-background-color: var(--ds-color-container-ui-tertiary-active-default, $ds-color-container-ui-tertiary-active-default);
+    }
+  }
+  
+}
+
+:host([variant='error']) {
+  --ds-auro-toast-background-color: var(--ds-color-container-error-default, #{$ds-color-container-error-default});
+  --ds-auro-toast-text-color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
+  --ds-auro-toast-icon-color: var(--ds-color-icon-error-default, $ds-color-icon-error-default);
+}
+
+:host([variant='success']) {
+  --ds-auro-toast-background-color: var(--ds-color-container-success-default, #{$ds-color-container-success-default});
+  --ds-auro-toast-text-color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
+  --ds-auro-toast-icon-color: var(--ds-color-icon-success-default, $ds-color-icon-success-default);
+}
+
+:host([visible][variant='error']),
+:host([visible][variant='success']) {
+  --ds-auro-toast-close-button-icon-color: var(--ds-color-text-primary-default, $ds-color-text-primary-default);
+}
```

Comment:
We should put line 44 in each of the specific selectors above. That makes it possible for someone to theme the button icon color for the two variants separately. (i.e. a different button color for each variant)